### ISSUE TITLE
Implement interactions when rendering panel pages

### ISSUE THREAD

**Issue by ng-druid** [rollthecloudinc/druid#343]
First Phase: https://github.com/rollthecloudinc/druid/issues/316

Interactions can be added to panel pages using the editor. However, these currently don't do anything. The interaction behavior needs to be implemented when rendering the panel page. There is some boilerplate code already inside the render components file to pass the interactions around between components like with the css and classes. However, nothing currently wires them up making them active. Some of the details of different approaches for that are outlined in the referenced first place issue.

**Comment by ng-druid** [comment]
The mvp has been merged already into 21 release.

**Comment by classifieds-dev** [comment]
Here’s a concise, appropriate reply you can use:

Thanks for the update. Since the MVP is already in the 21 release, I'll go ahead and diff the changes and cherry‑pick whatever is still relevant. We can run the remaining pieces through full review and address the functionality gaps you mentioned so that the final version is solid.

**Comment by ng-druid** [comment]
No, I mean I already did that and everything is good to go already in the 21 release.
